spec/lib/ansible/runner_spec.rb

```diff
@@ -17,6 +17,7 @@
   describe ".run" do
     let(:playbook) { "/path/to/my/playbook" }
     before do
+      allow(described_class).to receive(:wait_for).and_yield
```

Comment:
Errr... I pulled these directly from Adam's PR... but I think I need to remove the `.for_yield`.   Hold please.